doc/decisions/README.md

```diff
@@ -30,6 +30,7 @@ section here.
 - [Plugin Variants](plugin_variants.md)
 - [Ingroup Removal](ingroup_removal.md)
 - [Error Message Format](error_message_format.md)
+- [Binding implementation](binding_implementation.md)
```

Comment:
This name is way too general, your decision:

- is only about error codes
- not necessarily about bindings (you also talk about the macros in C)